packages/verkle/README.md

```diff
@@ -0,0 +1,111 @@
+# @ethereumjs/verkle
+
+[![NPM Package][verkle-npm-badge]][verkle-npm-link]
+[![GitHub Issues][verkle-issues-badge]][verkle-issues-link]
+[![Actions Status][verkle-actions-badge]][verkle-actions-link]
+[![Code Coverage][verkle-coverage-badge]][verkle-coverage-link]
+[![Discord][discord-badge]][discord-link]
+
+| Implementation of [Verkle Trees](https://ethereum.org/en/roadmap/verkle-trees/) as specified in [EIP-6800](https://eips.ethereum.org/EIPS/eip-6800) |
+| --------------------------------------------------------------------------------------------------------------------------------------------------- |
+
+> Verkle trees are a cryptographic data structure proposed for use in Ethereum to optimize storage and transaction verification. They combine features of Merkle Patricia Tries and Vector Commitment Trees to offer efficient data verification with smaller proof sizes. The goal is to improve scalability and efficiency in Ethereum's network operations.
+
+This package is currently in early alpha and is a work in progress. It is not intended for use in production environments, but rather for research and development purposes. Any help in improving the package is very much welcome.
+
+## Installation
+
+To obtain the latest version, simply require the project using `npm`:
+
+```shell
+npm install @ethereumjs/verkle
+```
+
+## Usage
+
+This class implements the basic [Modified Merkle Patricia Trie](https://ethereum.org/en/developers/docs/data-structures-and-encoding/patricia-merkle-trie/) in the `Trie` base class, which you can use with the `useKeyHashing` option set to `true` to create a trie which stores values under the `keccak256` hash of its keys (this is the Trie flavor which is used in Ethereum production systems).
+
+Checkpointing functionality to `Trie` through the methods `checkpoint`, `commit` and `revert`.
+
+It is best to select the variant that is most appropriate for your unique use case.
+
+### Initialization and Basic Usage
+
+```typescript
+import { VerkleTrie } from '@ethereumjs/verkle'
+
+const trie = new VerkleTrie()
+
+async function test() {
+  await trie.put(utf8ToBytes('test'), utf8ToBytes('one'))
```

Comment:
If `utf8ToBytes` and `bytesToUtf8` are used in the example it would be more clear to include them in the imports as well:
```
import { bytesToUtf8, utf8ToBytes } from 'ethereum-cryptography/utils.js'
```

Reply:
Agreed

Reply:
Done